lib/winston-logstash-udp.js

```diff
@@ -46,7 +46,13 @@ util.inherits(LogstashUDP, winston.Transport);
 winston.transports.LogstashUDP = LogstashUDP;
 
 LogstashUDP.prototype.connect = function() {
+    var self = this;
     this.client = dgram.createSocket('udp4');
+
+    this.client.on('error', function(err) {
+        // Skipping DNS errors
+        self.emit('error', err);
```

Comment:
I think I understand.  You're not trying to suppress the errors, you're trying to get them emitted up to a level that you can manage them at?